src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/EtwInput.cs

```diff
@@ -24,12 +24,17 @@ public class EtwInput : IObservable<EventData>, IDisposable, IRequireActivation
         private EventFlowSubject<EventData> subject;
         private ITraceEventSession session;
         private bool isDisposed;
+        private readonly string defaultSessionnamePrefix = $"EventFlow-{nameof(EtwInput)}";
```

Comment:
Make it `private static readonly string DefaultSessionNamePrefix` and move up next to `ProviderSectionName`